stories/components/peoplePicker/peoplePicker.properties.stories.js

```diff
@@ -80,6 +80,12 @@ closeModal.addEventListener('click', () => {
 
 export const disableSuggestions = () => html`
   <mgt-people-picker disable-suggestions></mgt-people-picker>
```

Comment:
Add some h1 titles for "disable suggestions" and "disable suggestions with default selected users" to distinguish the two.